oauthproxy_test.go

```diff
@@ -1015,6 +1016,22 @@ func TestRequestSignaturePostRequest(t *testing.T) {
 	assert.Equal(t, st.rw.Body.String(), "signatures match")
 }
 
+func TestDomain(t *testing.T) {
+	for i, tt := range []struct {
+		host, cookieDomain, sub, tld string
+	}{
+		{"internal.example.com", "<default>", "", "internal.example.com"},
+		{"internal.example.com", ".example.com", "internal", ".example.com"},
+		{"foo.internal.example.com", ".example.com", "foo.internal", ".example.com"},
+	} {
+		req := &http.Request{Host: tt.host}
+		p := OAuthProxy{CookieDomain: tt.cookieDomain}
+
+		sub, tld := p.domain(req)
+		assert.Equal(t, tt.sub+", "+tt.tld, sub+", "+tld, strconv.Itoa(i))
```

Comment:
Would it not be better to just assert `tt.sub == sub` and `tt.tld == tld`? This concatenation for me is kinda hard to read

Reply:
Fixed.